app/views/buyers/service_contracts/new.html.erb

```diff
@@ -11,11 +11,10 @@
       <p><%= h @service.name %></p>
     </li>
 
-    <%= form.input :plan, :collection => @service_plans, :include_blank => !@service_contract.plan_id? %>
```

Comment:
I'm just wondering why we did it like this and if something would break in an edge case if we stop doing it. You don't think so @Martouta ?

Reply:
Right, makes sense 👍 I am checking another PR now but I will check this later 👍

Reply:
I thought about having it some edge case. But currently the controller as far as I identified, it doesn't accept a blank service plan.

Reply:
so only if no service plan has been marked as default the empty option is rendered. that kind of makes sense to me but would be better to use `prompt: 'Select a plan` instead of include blank

Reply:
As I understand:

- it is on creation of service contract
- if there is already a plan pre-selected (I guess passed on parameters on the `new` action):
  *  We select this plan

- if there is no plan pre-selected:
  *  the default service plan should be selected as default if it exists
  * If there is no default service plan, then render a blank option


So I think to make it equal in every situation, we just `include_blank: true` and make sure that the above statements are right

Reply:
@hallelujah the problem is that `include_blank: true` will always render the blank option, which is not the desired behave. We wanted to render the blank option only if there is no default service plan. Also, @thomasmaas suggested to change it to `prompt`, to render the message instead of a blank option